README.md

```diff
@@ -5,3 +5,8 @@ A Python 3.6 implementation of the [Request Network API](https://github.com/Requ
 
 To contribute please see the Wiki!
 
+# Dependencies
+ipfsapi (pip install ipfsapi)
```

Comment:
there's a `requirements.txt` and you can just `pip install -r requirements.txt`. Also `asyncio` was added in Python 3.4 so you shouldn't need to install it.

Reply:
Thanks for clarifying this up!